searx/settings.yml

```
@@ -1548,6 +1548,24 @@ engines:
     page_size: 25
     disabled: true
 
+  - name: right dao
+    engine: xpath
+    paging: true
+    page_size: 12
+    search_url: https://rightdao.com/search?q={query}&start={pageno}
+    results_xpath: //div[@class="description"]
+    url_xpath: ../div[@class="title"]/a/@href
+    title_xpath: ../div[@class="title"]
```

```suggestion
    results_xpath: //div[contains(@class, "description")]
    url_xpath: ../div[contains(@class, "title")]/a/@href
    title_xpath: ../div[contains(@class, "title")]
```
prevents in from breaking in case they add additional classes to these html elements.